app/controllers/ApplicationController.java

```diff
@@ -20,6 +20,6 @@ public ApplicationController(final ControllerDependency controllerDependency) {
     }
 
     public F.Promise<Result> index() {
-        return F.Promise.pure(redirect(productcatalog.controllers.routes.ProductCatalogController.pop(1)));
+        return F.Promise.pure(redirect(productcatalog.controllers.routes.ProductOverviewPageController.pop(1)));
```

Comment:
Instead of `pop` (which I would like to avoid, because not everybody understands PDP and POP acronyms), maybe we can call it now `show` or something like that for the main action of the controller.